docs/azure/get-started-with-azure/guardrails.md

```diff
@@ -0,0 +1,220 @@
+# Azure landing zone guardrails: What you need to know
+
+Last updated: **{{ git_revision_date_localized }}**
+
+This document explains the "guardrails" (security and compliance rules) set up in our Azure Landing Zone. These guardrails are implemented using Azure Policy, and they are in place to ensure our cloud environment remains secure, compliant (including with standards like **CCCS Medium [Canada Federal PBMM]**), and cost-effective. They help us maintain consistency and prevent configurations that could lead to problems.
+
+Think of these guardrails as automatic checks and balances. They don't require you to do anything extra most of the time, but they might prevent you from doing something that goes against our established best practices, or they might automatically configure resources to meet compliance standards.
+
+!!! info
+    This document focuses on the things that you might be *prevented* from doing (Deny policies), the things you *must* do as a result, or things that will be automatically configured (DeployIfNotExists and Modify policies). Audit policies are also in place, which will flag non-compliant resources, but won't block deployments.
+
+## Key areas covered
```

Comment:
Should these bulleted headings also be links to their respective sub-sections for quicker navigation?